src/test/java/com/wikia/webdriver/common/dataprovider/SpecialPagesDataProvider.java

```diff
@@ -2,14 +2,9 @@
 
 import org.testng.annotations.DataProvider;
 
-/**
- * Bogna 'bognix' Knychala
- */
 public class SpecialPagesDataProvider {
 
-  private SpecialPagesDataProvider() {
-
-  }
+  private SpecialPagesDataProvider() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)